src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -276,21 +346,82 @@ export const MenuList = (props): JSX.Element => {
     )
 }
 
-export const customOption = (data: OptionTypeWithIcon, className = '') => {
+export const ToolsMenuList = (props): JSX.Element => {
+    const lastIndex = props.options?.length - 1
     return (
-        <div className={`flex left ${className}`}>
-            <img
-                src={MONITORING_TOOL_ICONS[data.label.toLowerCase()] || MONITORING_TOOL_ICONS.other}
-                alt={data.label}
-                style={{
-                    width: '20px',
-                    height: '20px',
-                    marginRight: '12px',
-                }}
-                onError={onImageLoadError}
-            />
-            <span className="dc__ellipsis-right">{data.label}</span>
-        </div>
+        <components.MenuList {...props}>
+            <>
+                {props.options ? (
+                    <div
+                        style={{
+                            display: 'grid',
+                            rowGap: '10px',
+                        }}
+                    >
+                        {props.options.map((_opt, idx) => (
+                            <Fragment key={_opt.label}>
+                                <div
+                                    style={{
```

Comment:
move these to css file and use classes instead

Reply:
Done